### ISSUE TITLE
Indices for createdAt and updatedAt attributes

### ISSUE THREAD

**Issue by pbespechnyi** [drudge/mongoose-timestamp#36]
Is there any way of defining indices for createdAt and updatedAt attributes?

**Comment by drudge** [comment]
I've added the ability to pass extra options (like index) when defining the fields. This is available in version 0.6.0 on npm.

Example:

```js
var opts = {
	createdAt: {
		name: 'customNameCreatedAt',
		type: String,
		index: true
	},
	updatedAt: {
		name: 'customNameUpdatedAt',
		type: String,
		es_indexed: true
	}
};

var CustomizedTypeOptionsTimeCopSchema = new Schema({
	email: String
});

CustomizedTypeOptionsTimeCopSchema.plugin(timestamps, opts);
CustomizedTypeOptionsTimeCopSchema.plugin(mongoosastic);
```

**Comment by pbespechnyi** [comment]
Thanks!
